docs/layouts/shortcodes/generated/expert_forst_section.html

```diff
@@ -0,0 +1,54 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-coordinator</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to let the task thread be the coordinator thread responsible for distributing requests.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.executor.inline-write</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Whether to let write requests executed within the coordinator thread.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.local-dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The local directory (on the TaskManager) where ForSt puts some meta files. Per default, it will be &lt;WORKING_DIR&gt;/tmp. See <code class="highlighter-rouge">process.taskmanager.working-dir</code> for more details.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.memory.fixed-per-slot</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>MemorySize</td>
+            <td>The fixed total amount of memory, shared among all ForSt instances per slot. This option overrides the 'state.backend.forst.memory.managed' option when configured.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.memory.fixed-per-tm</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>MemorySize</td>
+            <td>The fixed total amount of memory, shared among all ForSt instances per Task Manager (cluster-level option). This option only takes effect if neither 'state.backend.forst.memory.managed' nor 'state.backend.forst.memory.fixed-per-slot' are not configured. If none is configured then each ForSt column family state has its own memory caches (as controlled by the column family options). The relevant options for the shared resources (e.g. write-buffer-ratio) can be set on the same level (config.yaml).Note, that this feature breaks resource isolation between the slots</td>
```

Comment:
suggest : The fixed total amount of memory per Task Manager (cluster-level option), shared among all ForSt instances

This option only takes effect if neither 'state.backend.forst.memory.managed' nor 'state.backend.forst.memory.fixed-per-slot' are not configured. -> "This option only takes effect if  'state.backend.forst.memory.managed' and 'state.backend.forst.memory.fixed-per-slot' are not configured."

Reply:
Actually, these options are borrowed from the RocksDB as well as the descriptions. There were some unreasonable parts in the previous statement, I changed it.